open-irc-links/README.md

```diff
@@ -0,0 +1,13 @@
+# open-irc-links
+
+
+## What it does
+
+This add-on sets default client for opening IRC links using protocol_handlers.
+
+Whenever any link with IRC protocol is clicked it opens with pre-defined URI template.
```

Comment:
Please change to "Whenever a link using the IRC protocol is clicked, the link is opened in the URI defined in the add-on's manifest. In this example, all IRC protocol links are opened in mibbit.com."